File: glip-team-messaging/teams/create-team/code-samples/createGlipTeam.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// POST BODY
$body = array(
    'public' => true,
    'name' => '<ENTER VALUE>',
    'description' => '<ENTER VALUE>',
    'members' => array(
        array(
            'id' => '<ENTER VALUE>',
            'email' => '<ENTER VALUE>'
        )  
    )
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/glip/teams", $body);
// PROCESS RESPONSE
?>

File: glip-team-messaging/teams/create-team/code-samples/createGlipTeam.py
# https://developers.ringcentral.com/my-account.html#/applications
# Find your credentials at the above url, set them as environment variables, or enter them below

# POST BODY
body = {
    'public': true,
    'name': '<ENTER VALUE>',
    'description': '<ENTER VALUE>',
    'members': [
        {
            'id': '<ENTER VALUE>',
            'email': '<ENTER VALUE>'
        }
    ]
}

import os
from ringcentral import SDK
rcsdk = SDK(os.environ['clientId'], os.environ['clientSecret'], os.environ['serverURL'])
platform = rcsdk.platform()
platform.login(os.environ['username'], os.environ['extension'], os.environ['password'])
r = platform.post('/restapi/v1.0/glip/teams', body)
# PROCESS RESPONSE